server/controllers/api/documents.js

```diff
@@ -0,0 +1,177 @@
+/* eslint import/no-unresolved: 0 */
+import db from '../../models/';
+
+const docAttributes = (doc) => {
+  const attributes = {
+    id: doc.id,
+    title: doc.title,
+    content: doc.content,
+    OwnerId: doc.OwnerId,
+    isPublic: doc.isPublic,
+    tags: doc.tags,
+    createdAt: doc.createdAt,
+    updatedAt: doc.updatedAt
+  };
+
+  return attributes;
```

Comment:
You really dont need to assign this to a variable. you can just return the object. Eg.  
```JavaScript
return( {
              id: doc.id,
              title: doc.title,
              content: doc.content,
              ownerId: doc.OwnerId,
              isPublic: doc.isPublic,
              tags: doc.tags,
              createdAt: doc.createdAt,
              updatedAt: doc.updatedAt
           });
```